app/src/test/java/com/android/sample/model/profile/ProfileViewModelTest.kt

```diff
@@ -192,4 +195,33 @@ class ProfileViewModelTest {
     profileViewModel.clearUserData()
     assertEquals(null, profileViewModel.userState.value)
   }
+
+  @Test
+  fun fetchUserDataCachesLocallyOnSuccess() = runTest {
+    val mockUser = testUser // Replace with your test user object
```

Comment:
Passing a testuser from dummyData into a local variable in your test isn't really usefull